packages/cli/src/data/queries/menu.gql

```diff
@@ -0,0 +1,20 @@
+query($name: String, $route: String, $order: MenuOrderBy) {
+  menu(name: $name, pathname: $route, orderBy: $order) {
+    item {
+      label,
+      link
+    }
+    children {
+      item {
+        label,
+        link
+      },
+      children {
```

Comment:
will this need some logic to scale, or if I am reading this right, this only supports one level of nesting, right?  Maybe related to #273

Reply:
yes, only one level